packages/server/src/plugin/landingPage/default/index.ts

```diff
@@ -94,9 +101,16 @@ function ApolloServerPluginLandingPageDefault<TContext extends BaseContext>(
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'nonce-${nonce}' https://apollo-server-landing-page.cdn.apollographql.com/${version}/static/js/main.js https://embeddable-sandbox.cdn.apollographql.com/${encodeURIComponent(
```

Comment:
Should this version be encoded as well?

Reply:
When I left out the version it didn't work, I needed the whole path in there. 

I might be misunderstanding what you are suggesting